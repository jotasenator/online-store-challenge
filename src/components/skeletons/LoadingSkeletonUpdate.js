import React from 'react'
import Skeleton from 'react-loading-skeleton';

export default function LoadingSkeletonUpdate() {
    return (
        <div className='container mt-5 mb-5 pt-3 pb-3 '>
            <Skeleton count={6} height={60} />
            <div className='head-table-skeleton'>
                <div><Skeleton width={210} height={40} /></div>
            </div>
        </div>
    )
}
