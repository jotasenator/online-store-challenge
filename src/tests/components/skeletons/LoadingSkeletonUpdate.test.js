import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'

import LoadingSkeletonUpdate from '../../../components/skeletons/LoadingSkeletonUpdate'


describe('Snapshot from LoadingSkeletonUpdate', () => {

    test('should render <LoadingSkeletonUpdate/> correctly', () => {
        const wrapper = shallow(<LoadingSkeletonUpdate />)
        expect(wrapper).toMatchSnapshot()
    })

    test('should show me what is inside className=container', () => {
        const wrapper = shallow(<LoadingSkeletonUpdate />)
        expect(wrapper.find('.container').text()).toBe('<Skeleton /><Skeleton />')
    })
    test('should show me what is inside className=head-table-skeleton', () => {
        const wrapper = shallow(<LoadingSkeletonUpdate />)
        expect(wrapper.find('.head-table-skeleton').text()).toBe('<Skeleton />')
    })






})
