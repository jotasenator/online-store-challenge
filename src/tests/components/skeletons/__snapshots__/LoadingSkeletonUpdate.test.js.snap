// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot from LoadingSkeletonUpdate should render <LoadingSkeletonUpdate/> correctly 1`] = `
<div
  className="container mt-5 mb-5 pt-3 pb-3 "
>
  <Skeleton
    circle={false}
    count={6}
    delay={0}
    duration={1.2}
    height={60}
    width={null}
    wrapper={null}
  />
  <div
    className="head-table-skeleton"
  >
    <div>
      <Skeleton
        circle={false}
        count={1}
        delay={0}
        duration={1.2}
        height={40}
        width={210}
        wrapper={null}
      />
    </div>
  </div>
</div>
`;
